### ISSUE TITLE
Buildalyzer - AnalyzerManger ProjectFile-RequiresNetFramework property throws null exception for .vbproj

### ISSUE THREAD

**Issue by nandhinithirugnanam** [daveaglick/Buildalyzer#113]
Buildalyzer - AnalyzerManger ProjectFile-RequiresNetFramework property throws null exception for .vbproj
But it works for .csproj the exception occurs only for .vbproj.

> AnalyzerManager analyzerManager = new AnalyzerManager("C:\Users\\...\\source\\repos\\WebSite1\\WebSite1.sln");
> ProjectAnalyzer projectAnalyzer = analyzerManager.GetProject("D:\\vb2\\ConsoleApp1\\ConsoleApp1.vbproj");
> var analyzerResults = projectAnalyzer.Build();

For .csproj it works fine and gets value "RequiresNetFramework" property as false and its building the project as expected. But for .vbproj

>  projectAnalyzer.Build();

 throws null exception while building the project.

![image](https://user-images.githubusercontent.com/35485609/58607428-315b2f00-82bd-11e9-8355-dbe813525d71.png)

**Stack Trace:**
> at Buildalyzer.Construction.ProjectFile.<>c__DisplayClass14_0.<get_RequiresNetFramework>b__2(String i)
>    at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
>    at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
>    at Buildalyzer.Construction.ProjectFile.get_RequiresNetFramework()
>    at Buildalyzer.Environment.EnvironmentFactory.GetBuildEnvironment(String targetFramework, EnvironmentOptions options)
>    at Buildalyzer.ProjectAnalyzer.Build(String targetFramework)

Note: I am using Buildalyzer 2.3.0

**Comment by nandhinithirugnanam** [comment]
Able to fix the null exception by the below code change. 
```
 public bool RequiresNetFramework =>
            _projectElement.GetDescendants(ProjectFileNames.Import).Any(x => ImportsThatRequireNetFramework.Any(i => x.GetAttributeValue(ProjectFileNames.Project)**?.**EndsWith(i, StringComparison.OrdinalIgnoreCase) **== true**))
            || _projectElement.GetDescendants(ProjectFileNames.LanguageTargets).Any(x => ImportsThatRequireNetFramework.Any(i => x.Value.EndsWith(i, StringComparison.OrdinalIgnoreCase)))
            || ToolsVersion != null;
```

**Comment by daveaglick** [comment]
I think this is resolved in #207 courtesy of @cslong - at a minimum an `AnalyzerResult` is now created for Visual Basic projects (and tested via unit test). I'll close this issue since it's working as far as I can tell, but please open a new one if it looks like there are still problems with VB support (though my ability to help will be limited since I don't know the first thing about VB or VB projects).
